ci: Use an up to date Ghidra setup action

antoniovazquezblanco opened the pull request:
Disclaimer: I maintain the up to date github action to setup ghidra.

PD: I would recommend you enable dependabot because it seemed other actions were also out of date.

Thanks!

mahaloz commented:
Hi @antoniovazquezblanco, thanks for the PR. I've noticed in your changes that you changed the owner of the action we have been using, `er28-0652`, to yourself. Is there a reason for this?

antoniovazquezblanco commented:
Yep. I've tried to fix some bugs and update dependencies in the original repo but the author did not respond so I've forked the repo and started to maintain it myself. You can see the change history.

Among the changes:
* Up to date dependencies
* Fixed some bugs regarding fetching the "latest" version
* Added cache
* Allow to specify a token to avoid rate limiting

I am sure I forget about many others.

Please feel free to merge or close this pr if you are not interested.

Thanks!

mahaloz commented:
Got it. Thanks for the context. I'll try it out; thanks for maintaining an older project :).
